_docs_integrate/delete-an-attribute.md

```diff
@@ -0,0 +1,127 @@
+---
+# !!! Warning: Do not edit this file; any changes must be replicated in Excel !!!
+permalink: integrate/delete-an-attribute
+published: true
+title: "Delete an Attribute"
+type: scenario
+toc: true
+properties:
+  - id: SC055
+  - category: Handling data deletion
+  - description:
+  - customer: All
+  - component: integrate
+  - level: Advanced
+  - implementation status: OPEN
+  - documentation status: OPEN
+  - published:
+  - link: integrate/delete-an-attribute
+require:
+required_by:
+---
+
+The exact process of deleting a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) depends on the kind of LocalAttribute at hand.
+Besides creating [IdentityAttributes]({% link _docs_integrate/data-model-overview.md %}#identityattribute) for yourself, a common usage of enmeshed is to share them with a peer or to receive shared Attributes from a peer.
+These different cases have different demands that need to be taken into account, wanting to delete an Attribute.
+
+In general, you can only delete data from your own wallet and never from the peer's.
+So, you can delete peer shared Attributes, your copy of own shared Attributes and unshared RepositoryAttributes.
+However, if you want the peer to delete an Attribute you shared with them, you will need to request it, since the peer may have a valid reason to still keep it for a certain amount of time.
+
+## Request the deletion of own Attributes from peer
+
+Wanting to delete an Attribute owned by you from the peer technically describes the endeavor of reverting sharing the Attribute with them.
```

Comment:
- My interpretation of "...revert sharing the Attribute with them" would be that the data status before sharing the Attribute and after deleting the Attribute should be the same for both Identities involved. However, since the own shared Attribute remains as a kind of history after the peer shared Attribute has been deleted, for me this does not technically correspond to reverting the sharing of the Attribute with them. That's why I had considered omitting the "technically" in the sentence or rephrasing it in another way, but I'm fine with it keeping as it is if you think it's understandable and accurate enough. Perhaps you could then add a sentence emphasizing that the own shared Attribute and its predecessors are retained as a kind of **history** of with whom the Attribute was shared once the peer shared Attribute and its predecessors have been deleted.
- 👍

Reply:
I changed the wording from "reverting sharing" to "withdrawing to permission to use".